### ISSUE TITLE
RiotClient.GetStaticRealm does not work for some regions

### ISSUE THREAD

**Issue by sim1-dll** [aj-r/RiotNet#2]
Hi, 
I am facing a strange issue. **RiotClient.GetStaticRealm** is not working for some regions (I tested EUW and TR not working), while it works fine for other regions (I tried NA,BR,LAN). My developer account is on EUW so I think it is not related to the account settings.

``` Csharp
RiotClient client = new RiotClient(Region.EUW, riotApiKey);
staticRealm = client.GetStaticRealm(); //NOT OK

RiotClient client2 = new RiotClient(Region.NA, riotApiKey);
staticRealm = client2.GetStaticRealm(); //OK
```

The message I receive is the following:
_A REST request failed. Status code: 403_
_{"status": {"message": "Forbidden", "status_code": 403}}_

**Comment by aj-r** [comment]
Cool bug. It looks like it's sending the request to https://euw.api.pvp.net/... but it should be sending it to https://global.api.pvp.net/...

It seems to work anyways for NA, but not for EUW.

**Comment by aj-r** [comment]
Fixed in release 2.1.0
